packages/cli/src/commands/deploy/latest.ts

```diff
@@ -553,12 +553,8 @@ export default async function main(
       client
     );
 
-    if (
-      deployment instanceof Error &&
-      deployment.code === 'missing_project_settings'
-    ) {
+    if (deployment.code === 'missing_project_settings') {
```

Comment:
@TooTallNate @styfle  Remove `deploymente instanceof Error &&`, the current check should be enough :+1